tests/addresses.rs

```diff
@@ -40,3 +40,26 @@ async fn public_key_to_address() {
         "atoi1qzt0nhsf38nh6rs4p6zs5knqp6psgha9wsv74uajqgjmwc75ugupx3y7x0r".to_string()
     );
 }
+#[tokio::test]
+async fn mnemonic_address_generation() {
+    let iota = Client::builder().with_offline_mode().finish().await.unwrap();
+
+    let mnemnonic = "acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast";
```

Comment:
@Thoralf-M I think this should read from `tests/fixtures/test_vectors.json` so that the changes are reflected in every test if we made a change in the `test_vectors.json` file.

Reply:
We can change it, but then it shouldn't be in the Python folder

Reply:
But there are also changes in https://github.com/iotaledger/iota.rs/blob/9aef4a5e611087a5caa70dd0efe3e4e235080438/bindings/python/native/tests/fixtures/test_vectors.json, they shouldn't be needed then?

Reply:
I moved the test vectors which is specific to python binding to the global file, so it can be removed now.

Reply:
But in wallet.rs and not iota.rs?

Reply:
Also in iota.rs too. I changed that before and just forgot 😅

Reply:
@Thoralf-M Is this resolved ?

Reply:
Yes